x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/label/label.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiText, EuiTextTruncate, EuiToolTip, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { NODE_LABEL_HEIGHT, NODE_LABEL_WIDTH } from '../styles';
+
+export interface LabelProps {
+  text?: string;
+}
+
+export const Label = ({ text = '' }: LabelProps) => {
+  const [isTruncated, setIsTruncated] = React.useState(false);
+  const { euiTheme } = useEuiTheme();
+  return (
+    <EuiToolTip content={isTruncated ? text : ''} position="bottom">
```

Comment:
_nit_ I think we can remove the EuiTooltip when truncated, `EuiTextTruncate` is already handles that.